docs/specification/value-types.md

```diff
@@ -32,7 +32,14 @@ can be used to specify the expected value.
 
 ### Normal Real Value
 
-Normally distributed value parameterized by a real-valued mean and standard deviation
+Normally distributed value parameterized by a real-valued mean and standard deviation.
+
+A normal distribution is considered valid with a set of [Real Bounds](../attribute-templates#real-bounds)
+if the mean lies between the upper and lower bound; the width of the distribution
+is not considered.
+From a computational perspective, this means Normal Real Values are actually
```

Comment:
```suggestion
From a computational perspective, this means Normal Real values are actually
```

Reply:
I feel like `Value` is part of the proper noun, to whatever degree I'm referring to a proper noun, and should stay capital.  If we're being inconsistent, I'd migrate to this stylization.

Reply:
OK. I retract my suggestion.